### PR TITLE
added Georgian (ka) translation

### PR DESCRIPTION
**Pull request by GioMac**

### CODE REVIEW

**Comment by FoggyMtnDrifter** [comment]
Please update `config.toml`, `vercel.json`, and `layouts/partials/head/head.html` with your language. There are examples already in the files for you to reference.

**Comment by domingoruiz** [comment]
The language definition is missing in config.toml, vercel.json and in the javascript of layouts/partials/head/head.html

**Comment by FoggyMtnDrifter** [comment]
@GioMac Here's a wiki link for reference, as well: https://wiki.rockylinux.org/en/Web/Website/contrib-i18n

**Comment by GioMac** [comment]
@domingoruiz @ressonix thanks for doc
how do we set weight values? it's missing in doc, also, i think, it's better to include information about branch too (devel first?)

**Comment by FoggyMtnDrifter** [comment]
@GioMac I think the best way to do the weights is to look at the current total number of languages, +1 it, and if needed, one of us can change it if there would be conflicts on that front.

**Comment by FoggyMtnDrifter** [comment]
Regarding the branch, that's not a huge deal, we can edit the PR, but I'll definitely let Hayden know.

**Comment by fabaff** [comment]
Related #33

**Comment by FoggyMtnDrifter** [comment]
Converted to draft until the rest of the setup is finished. ;)

File: i18n/ka.toml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  [landing.p2.content]
+  other = "Rocky Linux არის საზოგადოებრივი კორპორატიული დონის ოპერაციული სისტემა, რომელიც 100%-ით თავსებადი უნდა იყოს წამყვან ამერიკულ კორპორატიულ Linux დისტრიბუტივთან, ვინაიდან შესაბამისმა ქვემდგომმა პარტნიორმა შეიცვალა მიმართულება. ამ მომენტში, პროექტი საზოგადოების მიერ ინტენსიური შემუშავების პროცესშია. Rocky Linux პროექტის წამყვანია გრეგორი კურცერი, რომელმაც დააფუძნა CentOS პროექტი. ამ მომენტში გამოცემის თარიღი დაზუსტებული არ არის. მათ ვისაც პროექტში თანამონაწილეობა სურს, შეუძლიათ ამ საიტზე შემოთავაზებული საკომუნიკაციო საშუალებები გამოიყენონ."
+
+[faq]
+  [faq.title]
+  other = "ხშირად დასმული კითხვები"
+
+  [faq.q]
+    [faq.q.downstream-partner-direction]
+    other = "რას ნიშნავს, \"ქვემდგომმა პარტნიორმა შეიცვალა მიმართულება?\""
```
Comment (SergeCroise) [inline i18n/ka.toml:73]: მიმართულება\\"?"

**Review by SergeCroise — CHANGES_REQUESTED** [review]

**Comment by hbjydev** [comment]
I'm now at a point where I don't want to accept any more i18n PRs, simply because it's going to become FAR too unmaintainable, so I'm going to prioritise #134 and close this PR.

**Pull request closed without merging**